source/_reference-g-s.html.md.erb

```diff
@@ -5336,6 +5357,31 @@ Represents individual user settings for a specific sheet.  User settings may be
 **id** | number | Id of the report, sheet, Sight (aka dashboard), or template from which the enclosing report, sheet, Sight, or template was created
 **type** | string | **report**, **sheet**, **sight**, or **template**
 
+### SummaryField Object
+
+     |     |     |
+-----|-----|-----|
+**id** | number | SummaryField Id
+**contactOptions** | [ContactOption](#contactoption-object)[] | Array of ContactOption objects to specify a pre-defined list of values for the column. Column **type** must be **CONTACT_LIST** 
+**createdAt** | timestamp | Time of creation
+**createdBy** | [User](#user-object) | User object containing name and email of the summaryField's author
+**displayValue** | string | Visual representation of cell contents, as presented to the user in the UI.  See [Cell Reference](#cell-reference).
+**format** | string | The format descriptor (see [Formatting](#formatting))<br/>Only returned if the **include** query string parameter contains **format** and this column has a non-default format applied to it.
+**formula** | string | The  formula for a cell, if set. NOTE: calculation errors or problems with a formula do not cause the API call to return an error code. Instead, the response contains the same value as in the UI, such as cell.value = "#CIRCULAR REFERENCE".
```

Comment:
Might be worth changing 
"cell.value = "#CIRCULAR REFERENCE"." 
to
"field.value = "#CIRCULAR REFERENCE".